jupyter_server/services/sessions/handlers.py

```diff
@@ -49,10 +50,12 @@ async def post(self):
 
         if "notebook" in model and "path" in model["notebook"]:
```

Comment:
Should this be checking if `"name"` exists in the 'notebook' stanza?

Reply:
```suggestion
        if "notebook" in model and "name" in model["notebook"]:
```